Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/CloudFormationTemplateFinder.cs

```diff
@@ -34,12 +34,28 @@ public string DetermineProjectRootDirectory(string sourceFilePath)
         }
 
         public string FindCloudFormationTemplate(string projectRootDirectory)
+        {
+            var templateAbsolutePath = DetermineCloudFormationTemplatePath(projectRootDirectory);
+
+            if (!_fileManager.Exists(templateAbsolutePath))
+                _fileManager.Create(templateAbsolutePath).Close();
+            
+            return templateAbsolutePath;
+        }
+
+        public bool DoesCloudFormationTemplateExist(string projectRootDirectory)
+        {
+            var templateAbsolutePath = DetermineCloudFormationTemplatePath(projectRootDirectory);
+            return File.Exists(templateAbsolutePath);
```

Comment:
should use IFileManager wrapper here.
```suggestion
            return _fileManager.Exists(templateAbsolutePath);
```

Reply:
Good catch